encore/slurm_queue.py

```diff
@@ -14,23 +14,28 @@ def __init__(self, job_id, job_directory, config = None):
         else:
             self.config = dict()
 
-    def get_batch_headers(self, model_plan):
-        mem_per_cpu = model_plan.get("mem_per_cpu", 6500)
-        cores_per_job= model_plan.get("mem_per_cpu", 56)
-
-        return ["#!/bin/bash", 
-           "#SBATCH --partition={}".format(self.config.get("QUEUE_PARTITION", "encore")),
-           "#SBATCH --job-name=gasp_{}".format(self.job_id), 
-           "#SBATCH --mem-per-cpu={}".format(mem_per_cpu), 
-           "#SBATCH --workdir={}".format(self.job_directory), 
-           "#SBATCH --cpus-per-task={}".format(cores_per_job),  
-           "#SBATCH --time=14-0", 
-           "#SBATCH --nodes=1",
-           "export OPENBLAS_NUM_THREADS=1"]
+    def get_batch_headers(self):
+        sbatch_headers = ["#!/bin/bash"]
```

Comment:
Headers is now a list to support optional addition of `SLURM_ACCOUNT`. This is required for ARC to do processing, but may not be present everywhere.